examples/flower_classifier/image_pyfunc.py

```diff
@@ -55,7 +55,9 @@ def __init__(self, graph, session, model, image_dims, domain):
         probs_names = ["p({})".format(x) for x in domain]
         self._column_names = ["predicted_label", "predicted_label_id"] + probs_names
 
-    def predict(self, input, params: Optional[Dict[str, Any]] = None):
+    def predict(
+        self, input, params: Optional[Dict[str, Any]] = None
+    ):  # pylint: disable=unused-argument
```

Comment:
```suggestion
    def predict(
        self,
        input,
        params: Optional[Dict[str, Any]] = None,  # pylint: disable=unused-argument
    ):
```

is preferred because:

1. It's clear what's unused.
2. Only mark `params` as unused.

Reply:
Black will auto format it to 
```
def predict(
        self, input, params: Optional[Dict[str, Any]] = None  # pylint: disable=unused-argument
    ):
```